src/org/usfirst/frc/team2335/robot/commands/AutoTurn.java

```diff
@@ -0,0 +1,49 @@
+package org.usfirst.frc.team2335.robot.commands;
+
+import org.usfirst.frc.team2335.robot.Robot;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+//TODO: Fine tune values
+
+public class AutoTurn extends Command
+{
+	private double _turnVal;
+	
+    public AutoTurn(int turnVal) 
+    {
+        _turnVal = turnVal > 0 ? 0.3 : -0.3;
```

Comment:
Those are ternary operators, it's basically a way or shortening if/else statements. In this case: if turnVal is greater than 0, _turnVal = 0.3, else: -0.3.